packages/cfengine/build.sh

```diff
@@ -0,0 +1,27 @@
+TERMUX_PKG_HOMEPAGE=https://cfengine.com/
+TERMUX_PKG_DESCRIPTION="CFEngine is a configuration management technology that helps systems administrators automate and ensure the availability, security and compliance of mission-critical applications and services."
+TERMUX_PKG_LICENSE="GPL-3.0"
+TERMUX_PKG_VERSION=(3.15.0-2
+                    3.15.0-2)
```

Comment:
The idea is to list multiple versions if you need them, but you don't need them.